### PR TITLE
Delete component route

### PR DESCRIPTION
**Pull request by DravitLochan**
Fixes issue #134 
deployment link : http://delete-account.surge.sh

image

### CODE REVIEW

File: public/200.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<!doctype html>
+<html lang="en">
+  <head>
```
Comment (rishiraj824) [inline public/200.html]: Please don't add this file here. It's not required.

Comment (DravitLochan) [inline public/200.html]: Where should I add this file then?

Comment (rishiraj824) [inline public/200.html]: Add 200.html in build folder after running `npm run build` so that the routes do not show 404.

**Comment by DravitLochan** [comment]
@rishiraj824 removed it

**Pull request merged**